src/methods/relative_partial_randomization.jl

```diff
@@ -0,0 +1,89 @@
+export RelativePartialRandomization, RelativePartialRandomizationAAFT
+
+"""
+    RelativePartialRandomization(α = 0.5)
```

Comment:
Nitpicking: We want to be consistent with style, so please leave a blank line between the function/type signature and the remainder of the docstring.